src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -607,8 +609,10 @@ TEST_F(CommandRequestImplTest, AppNotFound_HashUpdateNotExpected) {
 }
 
 TEST_F(CommandRequestImplTest, SendProviderRequest_ByServiceType) {
-  resumption::LastStateImpl last_state("app_storage_folder",
-                                       "app_info_storage");
+  auto last_state = std::make_shared<resumption::LastStateWrapperImpl>(
+      std::make_shared<resumption::LastStateImpl>("app_storage_folder",
```

Comment:
@YarikMamykin propose to declare a constants for that

Reply:
@AKalinich-Luxoft  fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/9feb1741ceff524d2cdbb89b03596e288741eff2